sdks/python/apache_beam/runners/dataflow/internal/apiclient.py

```diff
@@ -38,6 +38,7 @@
 
 from apitools.base.py import encoding
 from apitools.base.py import exceptions
+import six
```

Comment:
We dropped the dependency on `six` in Beam, you can use sys.version_info[0].

Reply:
Thank you.